tools/dep_updaters/update-eslint.sh

```diff
@@ -58,6 +58,7 @@ rm -rf ../node_modules/eslint
         --no-save \
         --omit=dev \
         --omit=peer \
+        --legacy-peer-deps \
```

Comment:
I'd prefer to wait for plugins to officially support ESLint v9.

Reply:
Here's the PR that adds support to babel-eslint: https://github.com/babel/babel/pull/16414

Reply:
Ok so I'll set it as blocked untill that pr lands? or I can just close it

Reply:
I see https://github.com/nodejs/node/pull/50714 is ready to land

Reply:
https://github.com/nodejs/node/pull/50714 is not directly related but I would like to land it before we do a new automated update.